packages/rpc/src/trust-api/TrustRPC.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { TrustRPC } from './TrustRPC';
+import { CoinType } from '@trustwallet/types/lib/CoinType';
+import { getEnv } from '../utils';
+
+describe('TrustRPC', () => {
+    let rpc: TrustRPC;
+    const coin = CoinType.cosmos;
+
+    beforeAll(() => {
+        require('dotenv').config({ path: __dirname + '/./.env' });
+        rpc = new TrustRPC(getEnv('TRUST_RPC_URL'));
+    });
+
+    it('should get Cosmos price', async () => {
+        const price = await rpc.getPrices(coin, 'USD');
```

Comment:
can add `USD` to `Currency` types?

Reply:
This one is just for tests. But yeah, I'l fix it.